docs/designs/supporting_pytorch.md

```diff
@@ -0,0 +1,316 @@
+# Supporting PyTorch
+
+## Introduction
+
+ElasticDL is an open-source distributed deep learning programming framework based
+on TensorFlow and Kubernetes. Considering that PyTorch is more widely used in academia,
+this project will support PyTorch.
```

Comment:
ElasticDL is an elastic and k8s native distributed deep learning framework. It targets to be designed to work with any DL framework smoothly, say, TF, PyTorch, etc. This document is going to show you the design of how it works with PyTorch.